terraform/lambdas.tf

```diff
@@ -179,10 +179,10 @@ resource "aws_lambda_function" "aggregation" {
   tags             = local.tags
   layers           = [
     module.lambda_layers.raster_analysis_arn,
-    data.terraform_remote_state.lambda-layers.outputs.py310_numpy_1264_arn,
-    data.terraform_remote_state.lambda-layers.outputs.py310_rasterio_138_arn,
-    data.terraform_remote_state.lambda-layers.outputs.py310_shapely_185_arn,
-    data.terraform_remote_state.lambda-layers.outputs.py310_pandas_153_arn,
+    data.terraform_remote_state.lambda-layers.outputs.py310_numpy_arn,
+    data.terraform_remote_state.lambda-layers.outputs.py310_rasterio_no_numpy_arn,
```

Comment:
I don't think that aggregation needs the rasterio layer, but we can fix that later.